Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -737,54 +737,29 @@ without the weights (left, 25563 vertices) and with the weights (right, 19936 ve
 replaced by a new implementation. In this case, the initialization detects
 all connected components in the 3D segmented image, and inserts points in
 the triangulation for each connected component.
 
-For the meshing, in the previous example (\ref Mesh_3/mesh_3D_image.cpp), we called `make_mesh_3()` as follows.
-
-\snippet Mesh_3/mesh_3D_image.cpp Meshing
-
-In the example \ref Mesh_3/mesh_3D_image_with_custom_initialization.cpp,
-that call is replaced by:
-  -# the creation of an empty `%c3t3` object,
-  -# a call to a non-documented function
-    `initialize_triangulation_from_labeled_image()` that inserts points in
-    the triangulation,
-  -# then the call to `refine_mesh_3()`.
-
-\snippet Mesh_3/mesh_3D_image_with_custom_initialization.cpp Meshing
-
-The code of the function `initialize_triangulation_from_labeled_image()` is
-in the non-documented header \ref
-CGAL/Mesh_3/initialize_triangulation_from_labeled_image.h. As it is
-undocumented and may be removed or modified at any time, if you wish to
-use it then you should copy-paste it to your user code. The code of that
-function is rather complicated. The following lines show how to insert new
-points in the `%c3t3` object, with the calls to
-`MeshVertexBase_3::set_dimension()` and
-`MeshVertexBase_3::set_index()`.
-
-\snippet CGAL/Mesh_3/initialize_triangulation_from_labeled_image.h insert initial points
-
-The value of `index` must be consistent with the possible values of
-`Mesh_domain::Index`. In \ref
-CGAL/Mesh_3/initialize_triangulation_from_labeled_image.h, it is
-constructed using the API of the mesh domain, as follows. First the functor
-`construct_intersect` is created
-
-\dontinclude CGAL/Mesh_3/initialize_triangulation_from_labeled_image.h
-\skip Construct_intersection construct_intersection =
-\until construct_intersection_object
+\snippet Mesh_3/mesh_3D_image_with_image_initialization.cpp Meshing
+
+The parameter `CGAL::parameters::initial_points_generator` is used.
+It expects a functor that returns a set of points for the mesh
+initialization (following the concept `InitialPointsGenerator_3`). The functor
+`Construct_initial_points_labeled_image` is used in this example.
+It constructs points using the API of the mesh domain, as follows.
+First, the functor `construct_intersection` is created:
+
+\snippet CGAL/Mesh_3/Construct_initial_points_labeled_image.h construct intersection
 then the `%Mesh_domain::Intersection` object (a `%tuple` with three
 elements) is constructed using a call to the functor `construct_intersection`
-\skip Intersection intersect =
-\until construct_intersection
-and eventually `%index` is the element \#1 of `%intersect`.
-\skipline get<1>(intersect)
+\snippet CGAL/Mesh_3/Construct_initial_points_labeled_image.h use construct intersection
+and eventually `%intersect_index` is the element \#1 of `%intersect`.
+\snippet CGAL/Mesh_3/Construct_initial_points_labeled_image.h get construct intersection.
```

Comment:
Overnight documentation build gave:
```
/home/cgal-testsuite/cgal_doc_build/CGAL-6.1-Ic-66/doc/Mesh_3/Mesh_3.txt:761: warning: block marked with [get construct intersection.] for \snippet should appear twice in file CGAL/Mesh_3/Construct_initial_points_labeled_image.h, found it 0 times
```

Just drop the `.` at the end.

Reply:
@janetournois So Doxygen does show errors, when the snippets are not used correctly. 😌